app/src/electron/windowStateKeeper.js

```diff
@@ -1,48 +0,0 @@
-// Simple module to help you remember the size and position of windows.
-// Can be used for more than one window, just construct many
-// instances of it and give each different name.
-
-import { app } from 'electron';
-import jetpack from 'fs-jetpack';
-
-export default function (name, defaults) {
-
-    var userDataDir = jetpack.cwd(app.getPath('userData'));
```

Comment:
It has been saving in userData, where is it saving now?

Reply:
this is handled by a new package I installed (electron-window-state), it's saving a file in the same location that has almost the same filename (window-state.json vs window-state-main.json previously)